data/data-pipeline/data_pipeline/config.py

```diff
@@ -12,6 +12,7 @@
 
 # set root dir
 settings.APP_ROOT = pathlib.Path(data_pipeline.__file__).resolve().parent
+settings.DATA_PATH = settings.APP_ROOT / "data"
```

Comment:
I moved this here b/c i needed to access this value in the utils, and I needed to remove a circular import between the ETL base class and the utils.